Resources/doc/reference/form_field_definition.rst

```diff
@@ -227,16 +227,23 @@ You can easily add a new ``galleryHasMedias`` row by defining one of these optio
                 ->add('enabled')
                 ->add('name')
                 ->add('defaultFormat')
-                ->add('galleryHasMedias', 'sonata_type_collection', array(), array(
-                    'edit' => 'inline',
-                    'inline' => 'table',
-                    'sortable' => 'position',
-                    'limit' => 3
+                ->add('galleryHasMedias', 'sonata_type_collection', array(
```

Comment:
We usually don't wait for someone to take on such a big burden and ask people to apply the boy scout rule : if you modify a piece of code, make it entirely right, in spite of consistency. Of course, if you want to contribute the big change yourself first, you can.